18F/identity-idp
# LG-6348: Redirect user to "Ready to verify" page on completion

**Why:** So that a user who chooses to verify their identity in-person can be given instructions on how to complete the process in person.

This is meant to implement a basic scaffolding for the page of LG-6348, with actual page content to be built as follow-on work.

**Testing Instructions:**

1. Navigate to http://localhost:3000
2. Sign in
3. Navigate to http://localhost:3000/verify
4. Complete proofing flow up to document capture
5. Use [error document](https://developers.login.gov/testing/#testing-identity-proofing) to trigger error result
6. Click troubleshooting link to proof in person
7. Complete proofing flow
8. Observe page shown in screenshot below at end of proofing flow 

**Screenshot:**

![image](https://user-images.githubusercontent.com/1779930/178356015-5c995b6b-0b3a-4f3f-8761-0bfea0aec3b1.png)
